DataFormats/Detectors/FIT/FV0/include/DataFormatsFV0/RecPoints.h

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file RecPoints.h
+/// \brief Definition of the FV0 RecPoints class
+
+#ifndef ALICEO2_FV0_RECPOINTS_H
+#define ALICEO2_FV0_RECPOINTS_H
+
+#include "CommonDataFormat/InteractionRecord.h"
+#include "CommonDataFormat/TimeStamp.h"
+#include "DataFormatsFV0/ChannelData.h"
+#include "CommonDataFormat/RangeReference.h"
+#include "DataFormatsFV0/BCData.h"
+#include <array>
+#include "Rtypes.h"
+#include <TObject.h>
+#include <gsl/span>
+
+namespace o2
+{
+namespace fv0
+{
+struct ChannelDataFloat {
+
+  int channel = -1;       // channel Id
+  double time = -20000;   // time in ps, 0 at the LHC clk center
+  double charge = -20000; // charge [channels]
+  int adcId = -1;         // QTC chain
+
+  ChannelDataFloat() = default;
+  ChannelDataFloat(int Channel, double Time, double Charge, int AdcId)
+  {
+    channel = Channel;
+    time = Time;
+    charge = Charge;
+    adcId = AdcId;
+  }
+
+  void print() const;
+
+  ClassDefNV(ChannelDataFloat, 1);
+};
+
+class RecPoints
+{
+
+ public:
+  enum TimeTypeIndex : int { TimeFirst,
+                             TimeGlobalMean,
+                             TimeSelectedMean };
+
+  o2::dataformats::RangeReference<int, int> ref;
```

Comment:
If you define the class (rather than struct) then better keep all data members as private with corresponding getters/setters. Also, interleaving data members and methods is not a good practice.

@AllaMaevskaya since this is a copy paste from FT0 code, could you please fix it also there?

Reply:
Agreed. This is a simple reshuffling of data members and doesn't involve modification of any other files. @AllaMaevskaya would you like me to do it also for FT0 or do you prefer to follow it yourself?